docs/reference/commandline/volume_create.md

```diff
@@ -47,4 +47,15 @@ Some volume drivers may take options to customize the volume creation. Use the `
 These options are passed directly to the volume driver. Options for
 different volume drivers may do different things (or nothing at all).
 
-*Note*: The built-in `local` volume driver does not currently accept any options.
+The built-in `local` driver accepts options similar to the `mount` command:
```

Comment:
Minor nit, but while updating the docs, can you indicate that no options are supported on Windows.